lib/kb_stringtie/Utils/StringTieUtil.py

```diff
@@ -350,8 +374,7 @@ def _generate_output_file_list(self, result_directory):
                              allowZip64=True) as zip_file:
             for root, dirs, files in os.walk(result_directory):
                 for file in files:
-                    if not (file.endswith('.DS_Store') or 
-                            os.path.basename(root) == 'merge_result'):
+                    if not file.endswith('.DS_Store'):
```

Comment:
The extra gffcmp data is in the merge results directory so we no longer want to exclude it.